lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -211,6 +215,14 @@ check_resource_details_key_and_value:
   element: &key_value
     selector:
       xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')]
+check_key_and_resource_link:
```

Comment:
![resourcenameandlink](https://user-images.githubusercontent.com/12692381/64511406-d3f06880-d316-11e9-9221-27b0a36c934f.png)

Reply:
Used to check resources like this